### PR TITLE
Rebrand updates

### PR DESCRIPTION
**Pull request by kellyjandrews**
Changes for moving from Nexmo to Vonage namespace.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,14 +16,14 @@ When you're ready to start coding, fork this repository to your own GitHub accou
...
-5. `npm compile`
-6. Uncomment the appropriate line in `examples/run-examples.js` for each example you want to run.
-7. In the `examples` folder, copy `example.env` to `.env`.
-8. Update `.env` with your API key and API secret.
-9. `node examples/run-examples.js`
-
-When reviewing PRs, and switching branches to compare running examples against feature branches, don't forget to run `npm compile` after switching branches. Because examples run from the `lib/` folder, and that folder is ignored by Git, you'll need to re-compile the source every time you switch a branch.
+1. `git clone https://github.com/vonage/vonage-node-sdk.git`
+1. `cd vonage-node-sdk`
+1. `git checkout BRANCH_NAME`
```
Comment (slorello89) [inline CONTRIBUTING.md:21]: `git checkout -b BRANCH_NAME` unless the branch already exists

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,14 +16,14 @@ When you're ready to start coding, fork this repository to your own GitHub accou
...
+1. `npm install`
+1. `npm compile`
```
Comment (slorello89) [inline CONTRIBUTING.md]: npm complains that `compile` isn't an npm command

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -16,14 +16,14 @@ When you're ready to start coding, fork this repository to your own GitHub accou
...
+1. Download the `https://github.com/Vonage/vonage-node-code-snippets` repository
```
Comment (slorello89) [inline CONTRIBUTING.md]: maybe have contributors download the snippets into a separate directory. If you're contributing you probably don't want to have the snippets in the same directory as the repo you're contributing to as they may accidentally get picked up in commits and cause a git pita that might be discouraging

File: CONTRIBUTING.md (hunk 2 above)
Comment (slorello89) [reply CONTRIBUTING.md]: is this supposed to be `npm run compile`?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -16,14 +16,14 @@ When you're ready to start coding, fork this repository to your own GitHub accou
...
-6. Uncomment the appropriate line in `examples/run-examples.js` for each example you want to run.
-7. In the `examples` folder, copy `example.env` to `.env`.
-8. Update `.env` with your API key and API secret.
-9. `node examples/run-examples.js`
-
-When reviewing PRs, and switching branches to compare running examples against feature branches, don't forget to run `npm compile` after switching branches. Because examples run from the `lib/` folder, and that folder is ignored by Git, you'll need to re-compile the source every time you switch a branch.
+1. `git clone https://github.com/vonage/vonage-node-sdk.git`
+1. `cd vonage-node-sdk`
+1. `git checkout BRANCH_NAME`
+1. `npm install`
```
Comment (slorello89) [inline CONTRIBUTING.md:22]: should be a numbered list:

1. 
2. 
3. 
4. 

etc. . .

File: src/CallsResource.js [hunk 5]
```diff
@@ -18,7 +18,7 @@ class CallsResource {
   /**
    * Creates a new CallsResource.
    *
-   * @param {Credentials} creds - Credentials used when interacting with the Nexmo API.
+   * @param {Credentials} creds - Credentials used when interacting with the VonageAPI.
```
Comment (slorello89) [inline src/CallsResource.js]: Here and in a bunch of places `VonageAPI` should be `Vonage API`

File: src/Credentials.js [hunk 6]
```diff
@@ -9,15 +9,15 @@ import HashGenerator from "./HashGenerator";
...
+ * @param {string} apiKey - A VonageAPI Key
+ * @param {string} apiSecret - A VonageAPI Secret
+ * @param {string} [applicationId] - A VonageApplication ID
  * @param {string|Buffer} [privateKey] -  When a string value is passed it should
  *                        either represent the path to the private key, or the actual
  *                        private key in string format. If a Buffer is passed then
  *                        it should be the key read from the file system.
- * @param {string} [signatureSecret] - A Nexmo signature Secret
- * @param {string} [signatureMethod] - A Nexmo compatible request signing method
+ * @param {string} [signatureSecret] - A Vonagesignature Secret
```
Comment (slorello89) [inline src/Credentials.js]: Looks like there was a find and replace issue where "Nexmo " was replaced with "Vonage"

File: src/DtmfResource.js [hunk 7]
```diff
@@ -12,7 +12,7 @@ class DtmfResource {
   /**
    * Creates a new DtmfResource.
    *
-   * @param {Credentials} creds - Credentials used when interacting with the Nexmo API.
+   * @param {Credentials} creds - Credentials used when interacting with the VonageAPI.
```
Comment (slorello89) [inline src/DtmfResource.js]: find and replace issue here as well

File: src/HttpClient.js [hunk 8]
```diff
@@ -246,7 +246,7 @@ class HttpClient {
     return function(err, data) {
       if (err && err.status == limitedAccessStatus) {
         err._INFO_ =
-          "This endpoint may need activating on your account. Please email support@nexmo.com for more information";
+          "This endpoint may need activating on your account. Please email support@vonage.com for more information";
```
Comment (slorello89) [inline src/HttpClient.js]: support@nexmo.com is still the correct email.

File: src/HttpClient.js [hunk 9]
```diff
@@ -39,7 +39,7 @@ class HttpClient {
 
     if (endpoint.method === "POST" || endpoint.method === "DELETE") {
       // TODO: verify the following fix is required
-      // Fix broken due ot 411 Content-Length error now sent by Nexmo API
+      // Fix broken due ot 411 Content-Length error now sent by VonageAPI
```
Comment (slorello89) [inline src/HttpClient.js]: missing space

File: src/FilesResource.js [hunk 10]
```diff
@@ -13,7 +13,7 @@ class FilesResource {
   /**
    * Creates a new FilesResource.
    *
-   * @param {Credentials} creds - Credentials used when interacting with the Nexmo API.
+   * @param {Credentials} creds - Credentials used when interacting with the VonageAPI.
```
Comment (slorello89) [inline src/FilesResource.js]: missing space

File: src/NumberInsight.js [hunk 11]
```diff
@@ -70,7 +70,7 @@ class NumberInsight {
    *                 Insight Return Parameters. This may be useful for your
    *                 internal reports.
    * @param {string} options['include-intermediate-callbacks'] - 'advanced' only.
-   *                 Tells the Nexmo platform to make callbacks as soon as an
+   *                 Tells the Vonageplatform to make callbacks as soon as an
```
Comment (slorello89) [inline src/NumberInsight.js]: missing space

File: src/StreamResource.js [hunk 12]
```diff
@@ -12,7 +12,7 @@ class StreamResource {
   /**
    * Creates a new StreamResource.
    *
-   * @param {Credentials} creds - Credentials used when interacting with the Nexmo API.
+   * @param {Credentials} creds - Credentials used when interacting with the VonageAPI.
```
Comment (slorello89) [inline src/StreamResource.js]: missing space

File: src/TalkResource.js [hunk 13]
```diff
@@ -12,7 +12,7 @@ class TalkResource {
   /**
    * Creates a new TalkResource.
    *
-   * @param {Credentials} creds - Credentials used when interacting with the Nexmo API.
+   * @param {Credentials} creds - Credentials used when interacting with the VonageAPI.
```
Comment (slorello89) [inline src/TalkResource.js]: missing space

File: src/Vonage.js [hunk 14]
```diff
@@ -48,10 +48,10 @@ class Nexmo {
       this.options.logger = new NullLogger();
     }
 
-    let userAgent = "nexmo-node/UNKNOWN node/UNKNOWN";
+    let userAgent = "@vonage/server-sdk/UNKNOWN node/UNKNOWN";
```
Comment (slorello89) [inline src/Vonage.js:51]: is there a way to set the `UNKNOWN`s to the correct sdk/language version? or am I just missing obvious?

File: test/Conversion-test.js [hunk 15]
```diff
@@ -64,7 +64,7 @@ describe("Conversion", function() {
       this.httpClientStub.request.yields(mockError, null);
       this.conversion.sms("1234", 1, 1234567890, function(err, data) {
         expect(err._INFO_).to.eql(
-          "This endpoint may need activating on your account. Please email support@nexmo.com for more information"
+          "This endpoint may need activating on your account. Please email support@vonage.com for more information"
```
Comment (slorello89) [inline test/Conversion-test.js]: support email is still `support@nexmo.com`

**Review by slorello89 — CHANGES_REQUESTED** [review]
Some questions about the contributor's guide, some emails are pointing at support@vonage.com (which I don't think actually route correctly yet). Looks like there as a problem with a find/replace that removed some important spaces.

File: CONTRIBUTING.md (hunk 4 above)
Comment (kellyjandrews) [reply CONTRIBUTING.md:22]: It already is ;)

File: CONTRIBUTING.md (hunk 3 above)
Comment (kellyjandrews) [reply CONTRIBUTING.md]: That was the sentiment - I'll be more explicit

File: src/Vonage.js (hunk 14 above)
Comment (kellyjandrews) [reply src/Vonage.js:51]: That's how it was when I got it - I have no idea why it's like that, but not changing it here

File: CONTRIBUTING.md (hunk 1 above)
Comment (kellyjandrews) [reply CONTRIBUTING.md:21]: Since this is specifically for a PR, wouldn't it already exist?

**Comment by kellyjandrews** [comment]
Ok @slorello89 thanks for catching those :). Should be a bit better now.

**Review by superdiana — COMMENTED** [review]
LGTM

**Pull request merged**